src/server/pbsd_main.c

```diff
@@ -1907,27 +1907,25 @@ main(int argc, char **argv)
 			pbs_scheduler_addr = get_hostaddr(pbs_conf.pbs_secondary);
 			if (contact_sched(SCH_SCHEDULE_NULL, NULL, pbs_scheduler_addr, pbs_scheduler_port) < 0) {
 				char **workenv;
-				char schedcmd[MAXPATHLEN+1];
-				/* save the current, "safe", environment.          */
-				/* reset the enviroment to that when first started */
-				/* this is to get PBS_CONF_FILE if specified       */
+				char schedcmd[MAXPATHLEN + 1];
+				/* save the current, "safe", environment.
+				 * reset the enviroment to that when first started
+				 * this is to get PBS_CONF_FILE if specified.*/
 				workenv = environ;
 				environ = origevp;
 #ifdef WIN32
-				strcpy(schedcmd, "net start pbs_sched");
-				sprintf(log_buffer, "starting scheduler: %s", schedcmd);
-				log_event(PBSEVENT_SYSTEM | PBSEVENT_FORCE,
-					PBS_EVENTCLASS_SERVER, LOG_CRIT,
-					msg_daemonname, log_buffer);
+				strncpy(schedcmd, sizeof(schedcmd), "net start pbs_sched");
```

Comment:
Either use a snprintf() or put a '\0' in the last character of schedcmd.  I'd go with the snprintf() myself, but it is up to you.